src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java

```diff
@@ -65,7 +66,7 @@ public FoundIndication scan(Run build) throws IOException {
         String currentfile = build.getLogFile().getName();
         BufferedReader reader = null;
         try {
-            reader = new BufferedReader(build.getLogReader());
+            reader = new BufferedReader(new InputStreamReader(build.getLogInputStream(), build.getCharset()));
```

Comment:
getLogReader isn't deprecated just getLogFile
I've opened another PR for it:
https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/104